xml/virt_docs.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="cha-virt-docs">
+ <title>Help and documentation</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  Both &xen; and &kvm; hosts include various sources of information and documentation,
+  many of which are already available your installed system. The various documentation
+  packages provide additional content beyond what is available in the base installation.
+ </para>
+ <variablelist>
+  <varlistentry>
+   <term>Integrated help and man pages for shell commands</term>
+   <listitem>
+    <para>
+     The various virtualization packages provide commands for managing many aspects
+     of the virtualization host. It is not possible or expected to remember all
+     options supported by these commands. Most commands include integrated 'help'
+     subcommands and man pages which describe the command and any supported options.
+    </para>
+   </listitem>
+  </varlistentry>
+  <varlistentry>
+   <term>Documentation in <filename>/usr/share/doc/packages</filename></term>
+   <listitem>
+     <para>
+      Many of the virtualization packages provide additional content in their
+      documentation sub-package. As an example, the libvirt-doc package contains
+      all of the documentation available at libvirt.org, plus sample code
+      demonstrating use of the libvirt C API.
```

Comment:
maybe we can add:

"You can find the libvirt-doc package documentation in /usr/share/doc/packages/" (XXX note I see only README.rst NEWS.rst AUTHORS.rst there XXX),

and you can find the full documentation from libvirt.org in /usr/share/doc/libvirt/html/ .

Examples, including XML VM configuration snippets, can be found in /usr/share/doc/libvirt/examples/.

Reply:
I reworked this paragraph a bit and think it addresses your comment in a more general way. But I'll let you be the judge of that once I push an updated version :-).